ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -1273,6 +1273,44 @@ def test_numpy_instance_truediv__(
     )
 
 
+@handle_frontend_method(
```

Comment:
E   TypeError: handle_frontend_method() missing 3 required keyword-only arguments: 'class_tree', 'init_tree', and 'method_name'

Reply:
Made the changes